crates/cdk/src/mint_url.rs

```diff
@@ -91,6 +91,12 @@ impl fmt::Display for MintUrl {
     }
 }
 
+impl From<Url> for MintUrl {
+    fn from(url: Url) -> Self {
+        Self(url.to_string())
```

Comment:
IMO it's better to use `MintUrl::from_str` here because it does some extra validation and processing.